### PR TITLE
FIX: Android APK missing from RC3

### PR DESCRIPTION
**Pull request by blckmn**

### CODE REVIEW

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by McGiverGim** [comment]
This is not the good solution. For Android the idea was generate the release apk with a private certificate and publish it at Play Store. 
We are finally releasing the Android version with a public certificate?

**Comment by blckmn** [comment]
> This is not the good solution. For Android the idea was generate the release apk with a private certificate and publish it at Play Store. We are finally releasing the Android version with a public certificate?

This is restoring functionality to where it was before. I am looking at getting the playstore certificate setup also.

**Pull request merged**
